src/libs/OptionsListUtils.js

```diff
@@ -398,7 +398,11 @@ function getLastMessageTextForReport(report) {
                 reportAction.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE &&
                 ReportActionUtils.isMoneyRequestAction(reportAction),
         );
-        lastMessageTextFromReport = ReportUtils.getReportPreviewMessage(iouReport, lastIOUMoneyReport, true);
+        // Whether parent report is a 1:1 report
+        const isChatReport = ReportUtils.isChatReport(report);
```

Comment:
No need to introduce new variable. inline is better

Reply:
Updated to inline.